execution/src/posix/execution.cc

```diff
@@ -139,8 +146,37 @@ namespace leatherman { namespace execution {
         return fs.st_mode & S_IXOTH;
     }
 
-    string which(string const& file, vector<string> const& directories)
+    bool isBuiltin(string const& file)
```

Comment:
can we use snake case for the function name? i.e. `is_builtin`
this seems to be the standard here